expenses-manager-backend/src/main/java/com/endava/expensesmanager/controller/UsersController.java

```diff
@@ -31,12 +31,18 @@ public ResponseEntity<List<Users>> getAllUsers() {
         return ResponseEntity.ok(users);
     }
 
-    @GetMapping("/{id}")
+    @GetMapping("/id/{id}")
```

Comment:
Why did you add in path to be /id/{id}?

Reply:
Aici am adăugat să fie /id/{id}, deoarece am creat o funcție care să caute user-ul după nume. Dacă aveam și /{id}, și {name}, primeam eroarea 500 la request, deoarece nu știa serverul pe care dintre cele două să o apeleze. 

Eu am nevoie de endpointul care caută după nume când fac login, pentru că pe frontend când se face login am numele utilizatorului

Reply:
In momentul de fata vad ca endpoint-ul de get by name are deja /name/{name}. Recomandarea mea este ca in momentul in care faci un endpoint de get by id atunci sa fie direct <path>/{id}, fara a mai adauga id in path